kahuna/public/js/components/gr-chips/gr-filter-chooser-chip.js

```diff
@@ -52,19 +52,16 @@ grFilterChooserChip.controller('grFilterChooserChipCtrl', function() {
                 // dropdown which blurred the current chip
                 $grChipsCtrl.setFocusedChip(filterChip);
             } else {
-                // Not a key, turn it into an 'any' filter and move focus next text
-                const anyFilterChip = {
-                    type: 'filter',
-                    filterType: chip.filterType,
-                    key: 'any',
-                    value: value
-                };
-                const nextText = {
-                    type: 'text',
-                    value: ''
-                };
-                $grChipsCtrl.replaceChip(chip, [anyFilterChip, nextText]);
-                $grChipsCtrl.setFocusedChip(nextText);
+                // Not a key, turn it back to text search
+                $grChipsCtrl.focusEndOfFirstChip();
+                chip.value = "FIELD NOT FOUND";
+                $grFilterChooserChipCtrl.notValid = true;
+
+                $timeout(() => {
+                    $grFilterChooserChipCtrl.removeSelf();
+                    $grChipsCtrl.items[0].value += (" " + value);
+                }, 800);
+
```

Comment:
Gives the user time to read the 'FIELD NOT FOUND' message (which is inside the chip) before said chip gets removed. 
Thoughts?